src/components/spinup_component/spinup.sh

```diff
@@ -89,8 +89,14 @@ run_spinup() {
     cd ${RunDirs}/spinup_run
 
     if ! "$isAWS"; then
-        # Load environment with modules for compiling GEOS-Chem Classic
-        source ${GEOSChemEnv}
+	if [ ! -f "${InversionPath}/${GEOSChemEnv}" ]; then
```

Comment:
see comment above

Reply:
These lines have been moved to run_imi.sh in https://github.com/geoschem/integrated_methane_inversion/pull/157/commits/9629c549521385b0709e1aa33c1c158082f6ddc7.